ddtrace/internal/debug.py

```diff
@@ -137,3 +138,101 @@ def collect(tracer):
         partial_flush_enabled=tracer._partial_flush_enabled,
         partial_flush_min_spans=tracer._partial_flush_min_spans,
     )
+
+
+def pretty_collect(tracer, color=True):
+    class bcolors:
+        HEADER = "\033[95m"
```

Comment:
There are a couple of packages that would make this easier to deal with, e.g. `termcolor` or `ansimarkup`, but it'd mean adding to the dependencies.

Reply:
Yeah, Kyle suggested I not use those so as to avoid adding dependencies.

Reply:
only problem I have with no dependency is... compatibility

how well does this work on windows? etc

Reply:
Does the `--colorless` flag which prints without adding the codes not cover this?

Reply:
👍🏻